doc/api/_toc.markdown

```diff
@@ -1,7 +1,10 @@
 @// NB(chrisdickinson): if you move this file, be sure to update tools/doc/html.js to
 @// point at the new location.
 * [About these Docs](documentation.html)
-* [Synopsis](synopsis.html)
+* [Synopsis & Example](synopsis.html)
```

Comment:
I'm not sure about that 
 - That file is not a synopsis of anything. 
 - Example doesn't really describe much.

What about "Hello World" or something like that?